packages/nextjs/hooks/scaffold-stark/__tests__/useAutoConnect.test.ts

```diff
@@ -0,0 +1,104 @@
+import { renderHook } from "@testing-library/react";
```

Comment:
Hi @danielcdz, we didnt mention this before since these tests are new features. For the challenges we dont need to migrate all this "__test__" folder

Reply:
Done! ✅